Include/internal/pycore_fileutils.h

```diff
@@ -74,14 +74,15 @@ extern int _Py_EncodeNonUnicodeWchar_InPlace(
     Py_ssize_t size);
 #endif
 
+extern PyAPI_FUNC(int) _Py_isabs(const wchar_t *path);
```

Comment:
Use `PyAPI_FUNC(int)` if you need it in stdlib modules built as shared library. Or use `extern` if not. But don't use both :-)